lms/djangoapps/instructor/static/instructor/ProblemBrowser/data/api/client.js

```diff
@@ -0,0 +1,34 @@
+import 'whatwg-fetch';
+import Cookies from 'js-cookie';
+
+const HEADERS = {
+  Accept: 'application/json',
+  'X-CSRFToken': Cookies.get('csrftoken'),
+};
+
+function initiateProblemResponsesRequest(endpoint, blockId) {
+  const formData = new FormData();
+    // xss-lint: disable=javascript-jquery-append
+  formData.append('problem_location', blockId);
```

Comment:
Nit: change to `formData.set('problem_location', blockId);` and you won't trip the XSS linter here.